### PR TITLE
live updates 🎉 

### PR DESCRIPTION
**Pull request by alangecker**
* Resolves: #256  
* Requires: #3876 
* Target version: master 

### Summary
out

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by matbgn** [comment]
@alangecker You are just sooooo amazing thanks a lot for your incredible job on #3876!

Just absolutely exited to see your achievement here also! And as promised as soon as you're near to release I will reward your job again.

Words are just out of my mind, so I will keep it simple : THANK YOU!

**Comment by matbgn** [comment]
@juliushaertl could you let run the CI to see what need to be done there?

**Comment by juliushaertl** [comment]
@alangecker I invited you as a contributor to the repo so follow up pushes should not require approval any longer.

**Comment by alangecker** [comment]
@juliushaertl ui, thanks! that helps a lot and probably also reduces your work load 🙂

**Comment by matbgn** [comment]
The functionality is there and all the checks in the CI are green. I mean, it's time for a reward!

I really hope that either @juliushaertl or @marcelklehr will have the time to review it soon, but for myself it's time to act 😄 

Again many thanks @alangecker and wish you an happy new year!

**Comment by matbgn** [comment]
Hi everyone, 

I know we all have many things on the plate, but we are eagerly waiting on this MR for our staff.

How do you see the next steps @juliushaertl and @marcelklehr?

Wish you all the best

**Comment by juliushaertl** [comment]
I've blocked some time to review and test this next week.

**Comment by marcelklehr** [comment]
Wow, I totally missed this in my inbox so far. Sorry!

File: src/sessions.js
Comment (marcelklehr) [inline src/sessions.js:64]: Without having checked myself: Would it be possible to only update the changed card?

**Comment by marcelklehr** [comment]
Code looks good IMHO. Will try to find time to test next week.

File: src/sessions.js
Comment (juliushaertl) [reply src/sessions.js:64]: This would be definitely nice if we could just fetch the one updated card

**Review by juliushaertl — APPROVED** [review]
Tested and works like a charm 👍 

Thanks for this awesome contribution. We can merge this from my side if rebased and the conflict is resolved.

**Comment by matbgn** [comment]
Amazing!!! 🍾 

If @alangecker fixes the very last details in the next few days, when do you plan to release the public version for Nextcloud users? If I'm not mistaken, #3876 is still not available with the last version 1.8.3, is it?

**Comment by alangecker** [comment]
thanks to all of you ♥

there are somehow now some tests failing.
I'll have a look into them during the next days and got already a draft for fetching only the updated card on the disk 🙂

**Comment by juliushaertl** [comment]
I'll check the psalm errors as they also happen on master. I've noticed a small bug when trying to share a board, the fix was straight forward but maybe you can put that into your branch directly so we have the changes together: https://github.com/nextcloud/deck/pull/4452/commits/49d424e470ef4a1af32fd94ac8d6091d3c7e446e

**Comment by juliushaertl** [comment]
psalm checks should also pass now after a rebase :)

**Comment by small1** [comment]
Any updates on this?

**Comment by juliushaertl** [comment]
See https://github.com/nextcloud/deck/pull/4273#issuecomment-1429799183

**Comment by juliushaertl** [comment]
I took the liberty to rebase and push so we can get this in for the first pre-release for Nextcloud 26.

**Comment by juliushaertl** [comment]
Let's get this in 🚀

**Comment by szaimen** [comment]
🎉🎉🎉🎉🎉

**Pull request merged**
